dockerfiles/alpine/micro-integrator/Dockerfile

```diff
@@ -53,10 +54,11 @@ RUN \
 # copy init script to user home
 COPY --chown=wso2carbon:wso2 docker-entrypoint.sh ${USER_HOME}/
 # install required packages
-RUN apk add --no-cache netcat-openbsd
+RUN apk add --no-cache netcat-openbsd \
```

Comment:
@NishikaDeSilva shall we move the `netcat-openbsd` to a new line as in SI's Alpine Dockerfile?